src/controllers/group/index.js

```diff
@@ -0,0 +1,5 @@
+import { group } from '~/src/models'
+
+export const getGroups = () => group.findAll()
+
+export const getProduct = (obj, args, context) => obj.getProduct()
```

Comment:
You can ignore args and context here, since you're only using "obj".

Reply:
Fixed